### PR TITLE
Fixed floor_divide deprecation warnings seen in pytest output

### PR DESCRIPTION
**Pull request by prabhat00155**
Addresses https://github.com/pytorch/vision/issues/3668

### CODE REVIEW

File: torchvision/models/detection/roi_heads.py [hunk 1]
```diff
@@ -261,14 +261,11 @@ def heatmaps_to_keypoints(maps, rois):
...
         roi_map = F.interpolate(
             maps[i][:, None], size=(roi_map_height, roi_map_width), mode='bicubic', align_corners=False)[:, 0]
-        # roi_map_probs = scores_to_probs(roi_map.copy())
         w = roi_map.shape[2]
         pos = roi_map.reshape(num_keypoints, -1).argmax(dim=1)
 
         x_int = pos % w
-        y_int = (pos - x_int) // w
-        # assert (roi_map_probs[k, y_int, x_int] ==
-        #         roi_map_probs[k, :, :].max())
```
Comment (NicolasHug) [inline torchvision/models/detection/roi_heads.py:271]: These might have been intended as some kind of documentation - perhaps leave them here? They were introduced in https://github.com/pytorch/vision/pull/898

**Review by NicolasHug — COMMENTED** [review]
Thanks @prabhat00155 , looking at the logs it seems that there's still one warning showing: test/test_datasets.py::CelebATestCase::test_attr_names

**Comment by prabhat00155** [comment]
> Thanks @prabhat00155 , looking at the logs it seems that there's still one warning showing: test/test_datasets.py::CelebATestCase::test_attr_names

Thanks, I didn't see this warning locally, perhaps the test got skipped.

File: torchvision/models/detection/roi_heads.py (hunk 1 above)
Comment (prabhat00155) [reply torchvision/models/detection/roi_heads.py:271]: I see a few other commented out code in the same PR, for example:  `# print('creating index...')`, I assume these would have been for testing locally. I am not sure if it's a good idea to have commented out code checked-in. Anyways, I'll leave it here, perhaps we should revisit it later.

**Review by NicolasHug — APPROVED** [review]
Thanks!

**Pull request merged**